assets/blocks/reader-registration/edit.js

```diff
@@ -203,31 +203,29 @@ export default function ReaderRegistrationEdit( {
 				{ editedState === 'initial' && (
 					<div className={ `newspack-registration ${ className }` }>
 						<form onSubmit={ ev => ev.preventDefault() }>
+							<div className="newspack-registration__have-account">
```

Comment:
At smaller viewports, the close button overlaps the message, which looks unintentional:

<img width="367" alt="Screen Shot 2022-09-21 at 2 26 02 PM" src="https://user-images.githubusercontent.com/2230142/191603614-e2f0461b-12d1-4db5-b447-3b708d4c2c74.png">

I'd expect the message to appear full-bleed on top of the modal, similar to how it looks in the auth modal:

<img width="373" alt="Screen Shot 2022-09-21 at 2 27 38 PM" src="https://user-images.githubusercontent.com/2230142/191603865-63845f1f-fc3f-4647-add1-831da4714ae3.png">

Reply:
Fixed in b675cecf6d71696f371af7165b938f82d0596717